docs/getting-started/tutorials/load-notion-postgres.md

```diff
@@ -0,0 +1,101 @@
+However, when you want to leverage this data for analytics, reporting, or integration with other systems, you often need to move it into a structured database like PostgreSQL.
+That’s where Bruin comes in. Bruin makes it easy to automate and manage data pipelines, letting you move data from Notion (and other sources) to destinations like PostgreSQL with minimal effort.
+
+If you’re just starting out, don’t worry—this guide is designed for beginners. You’ll learn the basics of setting up a Bruin project, creating a pipeline,
+and connecting Notion with PostgreSQL. By the end, you’ll have a working example that you can extend or customize for your own needs.
+
+### Prerequisites
+#### Bruin CLI
+
+To install Bruin CLI depending upon your machine, follow the installation instructions [here](../introduction/installation.md)
+
+#### Bruin VSCode Extension
+To install bruin vscode extension, follow the instruction [here](../../vscode-extension/installation.md)
+
+<img width="685" alt="bruin_extension" src="https://github.com/user-attachments/assets/c74607e8-b822-4927-a100-bfe29446101b">
+
+
+### Getting Started
+
+To create a bruin project basic structure you can just run
+
+
+   ```
+   bruin init {folder name} [template name]
+   ```
+
+If you don't define folder name and template name. It will create `default` template with folder name bruin-pipeline. This command will:
+
+    Create a project named bruin-pipeline 
+    Generate a folder called bruin-pipeline containing the following:
+        -  An assets folder (where you add asset file)
+        - .bruin.yml file (where connection and credentials are added)
+        -  pipeline.yml file to manage your pipeline
+
+#### Adding a new asset
+
+Adding a new [asset](../../assets/definition-schema.md) is as simple as creating a new file inside the assets folder. Let's create a new [ingestr asset](../../assets/ingestr.md) file `asset.ingetsr.notion.yml` inside assets folder and add :
+
+ ```
+name: public.notion
+type: ingestr
+connection: my-postgres
+parameters:
+  source_connection: my-notion
+  source_table: 'database_id'
+  destination: postgres
+ ```
+
+- name: The name of the asset.
+
+- type: Specifies the type of the asset. It will be always ingestr type for Notion.
+
+- connection: The name of the connection
+
+- source_connection: The name of the Notion connection defined in .bruin.yml.
+- source_table: The database id of Notion you want to ingest.
+- destination: The name of the destination, you want to store. Here, we are using postgres sql
+
+
+#### Adding connection and credentials in .bruin.yml
+Make sure, you have [Notion credentials](https://dlthub.com/docs/dlt-ecosystem/verified-sources/notion#setup-guide) and Postgres credentials.
+
+Using Bruin vscode extension
+- Add destination connection - PostgresSQL
+- Add source connection - Notion
+
+<video width="685" height="auto" controls poster="https://github.com/user-attachments/assets/7f9bac4d-a8d4-4bbf-b114-948ca561f50b">
```

Comment:
we shouldn't use this, instead let's use relative links here with the files actually being there